img/logo-mobile-header.svg

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
All SVGs need to be optimized (compressed), there's too much meta data, please do this using [SVGOMG](https://jakearchibald.github.io/svgomg/)